scripts/ci/requirements.txt

```diff
@@ -14,7 +14,7 @@ flake8
 flask
 flask-admin
 flask-cache
-flask-login
+flask-login==0.2.11
```

Comment:
LGTM, but the real solution is to make it comaptible with both version by wrapping the calls to current_user.is_authenticated() in a try except, and also try the property, finally default to False.